packages/big-design/src/components/Worksheet/Row/Row.tsx

```diff
@@ -67,7 +67,7 @@ const InternalRow = <T extends WorksheetItem>({ columns, rowIndex }: RowProps<T>
           rowIndex={rowIndex}
           type={column.type ?? 'text'}
           validation={column.validation}
-          value={row[column.hash]}
+          value={row[column.hash] ?? ''}
```

Comment:
Isn't this an issue with `renderedCell` in Cell`? I see references to `renderedValue` taking care of this issue but it's not being used for every cell type...

Reply:
Let me take a look

Reply:
`renderedValue` in `Cell` is only used for `Text/Number` columns when they're not being edited, you're correct, all the other cell types handle that on their own. It is way easier for me to just pass in a default value than to update all the "rendered values" for all cell types.

Reply:
Will make some changes, though, since we get a default value.